config/ftbquests/quests/chapters/creative.snbt

```diff
@@ -0,0 +1,817 @@
+							}
+							Enchantments: [{ }]
+						}
+					}
+				}
+				{
+					id: "517C037090107437"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 5564196992594175882L
+				}
+			]
+		}
+		{
+			title: "Master of Dragons"
+			x: -3.0d
+			y: 1.0d
+			shape: "diamond"
+			size: 2.0d
+			id: "3615BAFD4F2AD479"
+			tasks: [{
+				id: "48CE28F5C1683093"
+				type: "item"
+				item: "allthetweaks:dragon_soul"
+			}]
+			rewards: [
+				{
+					id: "7952CD1C4621CA26"
+					type: "item"
+					item: "reliquary:pedestals/passive/white_passive_pedestal"
+				}
+				{
+					id: "26B9DA9F21EE83C4"
+					type: "item"
+					item: {
+						id: "allthemodium:allthemodium_sword"
+						Count: 1b
+						tag: {
+							display: {
+								Name: "[{\"text\":\"Master of Dragons\",\"italic\":false,\"color\":\"gold\"}]"
+							}
+							Enchantments: [{ }]
+						}
+					}
+				}
+				{
+					id: "17722771164A4D2C"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 5564196992594175882L
+				}
+			]
+		}
+		{
+			title: "Master of The Undead"
+			x: -2.0d
+			y: 6.0d
+			shape: "diamond"
+			size: 2.0d
+			id: "5477FBAEBDE2D10B"
+			tasks: [{
+				id: "042808E2D5E02DED"
+				type: "item"
+				item: "allthetweaks:withers_compass"
+			}]
+			rewards: [
+				{
+					id: "5241889A5D8B8931"
+					type: "item"
+					item: "reliquary:pedestals/passive/white_passive_pedestal"
+				}
+				{
+					id: "180F365B4EDFF409"
+					type: "item"
+					item: {
+						id: "allthemodium:allthemodium_sword"
+						Count: 1b
+						tag: {
+							display: {
+								Name: "[{\"text\":\"Master of The Undead\",\"italic\":false,\"color\":\"gold\"}]"
+							}
+							Enchantments: [{ }]
+						}
+					}
+				}
+				{
+					id: "62D680E24CAAD6F4"
+					type: "random"
+					title: "Random Mythic Reward"
+					table_id: 5196609362437981520L
+				}
+			]
+		}
+		{
+			title: "Infinite Mana"
+			x: -4.0d
+			y: -1.0d
+			shape: "hexagon"
+			hide_dependency_lines: true
+			dependencies: ["464D0C17601E8A2B"]
+			hide: true
+			size: 1.5d
+			id: "5C7B81756CA58056"
+			tasks: [
+				{
+					id: "775BD503F830BB6C"
+					type: "item"
+					item: "botania:creative_pool"
+				}
+				{
+					id: "7FDBB3698C087D8B"
+					type: "item"
+					item: {
+						id: "botania:mana_tablet"
+						Count: 1b
+						tag: {
+							mana: 500000
+							creative: 1b
+						}
+					}
+				}
+			]
+			rewards: [
+				{
+					id: "753A05B6079F4FBA"
+					type: "item"
+					item: "botania:gaia_head"
+				}
+				{
+					id: "7D74C1F49E9E5D28"
+					type: "xp_levels"
+					xp_levels: 100
+				}
+			]
+		}
+		{
+			title: "Set For Life"
+			x: 0.0d
+			y: 12.0d
+			shape: "hexagon"
+			subtitle: "Enough to make everything in the pack?"
+			description: ["If you make 9 Stars, or enough to make an ATM Star Block, you'll be able to create a &dStarry Bee&r. This bee gives you &6ATM Star Shards&r from their combs."]
```

Comment:
Last time I checked, the Starry Bee costs 1 ATM Star Block to create and requires another ATM Star Block to use as its flower, a total of 2 ATM Star Blocks (18 ATM Stars), not 1 block (9 stars).

Reply:
Forgot to change this. Thanks for pointing this out!